docs/ref/settings.txt

```diff
@@ -2201,6 +2201,19 @@ already have it.
 
     In older versions, the default value is ``False``.
 
+.. setting:: SECURE_FEATURE_POLICY
+
+``SECURE_FEATURE_POLICY``
+-------------------------
+
+.. versionadded:: 3.0
+
+Default: ``None``
+
+If configured, the :class:`~django.middleware.security.SecurityMiddleware` sets
+the :ref:`feature-policy` header on all responses that do not already have it
+to the value provided.
```

Comment:
This is missing a description of the expected format of the value (a directive -> allowed-list dict).

Reply:
Yes, I know - this is a work in progress!